sql/2018-04-17-subscribers-table.sql

```diff
@@ -0,0 +1,14 @@
+-- This file creates a new table, subscriptions, in the ejabberd DB.
+
+USE ejabberd;
+
+CREATE TABLE subscription (
```

Comment:
No id?

Reply:
None of the other ejabberd tables have id, and I was following that convention.  Is that okay?